Makefile.refactor.make

```diff
@@ -48,7 +48,7 @@ clean-glide:
 
 .PHONY: cleanall-glide
 cleanall-glide: clean-glide
-	@rm -rf .glide
+	@rm -rf .glide glide.lock
```

Comment:
@ndegory Yes, I'm aware. When I was testing, I wanted a way to remove glide.lock as part of make without having to rm manually; since it is an autogenerated file, it goes with the philosophy of cleanall to remove all of our generated artifacts. After a cleanall, I intentionally want `glide.lock` to be updated with a new time stamp. Without `glide.lock` being present, this forces glide to effectively do an update, which will recreate `glide.lock` pinned to the latest versions that satisfy the requirements of `glide.yaml`.